### PR TITLE
Added ping color

### PR DESCRIPTION
**Pull request by Bowhza**
image

### CODE REVIEW

File: H2MLauncher.UI/PingColorConverter.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using System.Windows.Documents;
+using System.Windows.Media;
+
+namespace H2MLauncher.UI;
+
+public class PingColorConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        long ping = (long)value;
```
Comment (metalglove) [inline H2MLauncher.UI/PingColorConverter.cs:12]: I recommend doing with pattern matches

```cs
if (value is long num) 
{
    // code here
    // return pingColor
}

throw new ArgumentNullException(nameof(value));// this should never happen

File: H2MLauncher.UI/MainWindow.xaml [hunk 2]
```diff
@@ -46,6 +46,7 @@
             </Style.Triggers>
         </Style>
         <local:HostNameColorConverter x:Key="hostNameColorConverter" />
+        <local:PingColorConverter x:Key="PingColorConverter"/>
```
Comment (metalglove) [inline H2MLauncher.UI/MainWindow.xaml:49]: We named the other converter `hostNameColorConverter` I think we can stick to that same style, so do `pingColorConverter`

File: H2MLauncher.UI/MainWindow.xaml [hunk 3]
```diff
@@ -190,6 +191,9 @@
                         <DataGridTextColumn.ElementStyle>
                             <Style TargetType="TextBlock">
                                 <Setter Property="HorizontalAlignment" Value="Right" />
+                                <Setter Property="FontFamily" Value="Consolas" />
+                                <Setter Property="FlowDirection" Value="RightToLeft" />
+                                <Setter Property="Foreground" Value="{Binding Ping, Converter={StaticResource PingColorConverter}}" />
```
Comment (metalglove) [inline H2MLauncher.UI/MainWindow.xaml:196]: See previous comment -> `pingColorConverter`

**Review by metalglove — CHANGES_REQUESTED** [review]

File: H2MLauncher.UI/MainWindow.xaml (hunk 2 above)
Comment (Bowhza) [reply H2MLauncher.UI/MainWindow.xaml:49]: image
I did it uppercase because of suggestion

Comment (metalglove) [reply H2MLauncher.UI/MainWindow.xaml:49]: Ah, ok. Then you might as well update the other one.

**Review by metalglove — APPROVED** [review]

**Pull request merged**